modules/apps/portal-workflow/portal-workflow-instance-tracker-web/src/main/java/com/liferay/portal/workflow/instance/tracker/web/internal/portlet/InstanceTrackerPortletProvider.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.portal.workflow.instance.tracker.web.internal.portlet;
+
+import com.liferay.portal.kernel.portlet.BasePortletProvider;
+import com.liferay.portal.kernel.portlet.ViewPortletProvider;
+import com.liferay.portal.workflow.constants.WorkflowPortletKeys;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Feliphe Marinho
+ */
+@Component(
+	immediate = true,
+	property = "model.class.name=" + WorkflowPortletKeys.INSTANCE_TRACKER,
+	service = ViewPortletProvider.class
+)
+public class InstanceTrackerPortletProvider
```

Comment:
Hey @MarinhoFeliphe,

I think this class is not required since we can access the `WorkflowPortletKeys.INSTANCE_TRACKER` directly.

Reply:
And the `PortletProvider` is used to map a className(an asset representation like com.*.CommerceOrder) to a Portlet that is responsible to handle some action(View, Add, Browser, Manage and etc).